lib/Basics/threads-posix.h

```diff
@@ -26,6 +26,9 @@
 
 #include "Basics/Common.h"
 
+// Compatible wth the Windows definition.
+#define INFINITE 0xFFFFFFFF // Infinite timeout
```

Comment:
Do we need to define `INFINITE` for _WIN32 as well?
I think it is not defined in our code. Is it predefined by some Windows header file?

Reply:
Windows comes with its own definition of `INFINITE` in `WinBase.h` (identical to this one).

Reply:
Ok, that I didn't know.